packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -90,16 +115,23 @@ export interface TerminalConfiguration {
     'terminal.integrated.fontFamily': string
     'terminal.integrated.fontSize': number
     'terminal.integrated.fontWeight': FontWeight
-    'terminal.integrated.fontWeightBold': FontWeight
+    'terminal.integrated.fontWeightBold': FontWeight,
+    'terminal.integrated.drawBoldTextInBrightColors': boolean,
     'terminal.integrated.letterSpacing': number
     'terminal.integrated.lineHeight': number,
     'terminal.integrated.scrollback': number,
+    'terminal.integrated.fastScrollSensitivity': number,
     'terminal.integrated.rendererType': TerminalRendererType,
     'terminal.integrated.copyOnSelection': boolean,
+    'terminal.integrated.cursorBlinking': boolean,
+    'terminal.integrated.cursorStyle': CursorStyleVSCode,
+    'terminal.integrated.cursorWidth': number
```

Comment:
One thing I noticed was, the preferences are only used when the terminal was initialized, which means the change of prefs wouldn't have any impact on the terminal instances that are already open. So i guess we could improved this change by subscribing the terminal pref updates.

Reply:
@elaihau in my case, for most terminal preferences I see that changes are applied to already existing terminals as well. For example, when changing the `terminal.integrated.cursorStyle` (the cursor successfully changes from `block` to `underline`:

<div align='center'>

![a](https://user-images.githubusercontent.com/40359487/76173054-429d6a00-6172-11ea-8523-d2980e9927a1.gif)

</div>

Reply:
> @elaihau in my case, for most terminal preferences I see that changes are applied to already existing terminals as well.

You said "for **most** terminal preferences" - does that mean changes to "some preferences being added in this PR" are not supposed to be applied to existing terminal instances?

I tested one more time in gitpod 

![Peek 2020-03-08 23-42](https://user-images.githubusercontent.com/37082801/76181345-219b4000-6197-11ea-8ff6-6c9f15a81a78.gif)


![Peek 2020-03-08 23-43](https://user-images.githubusercontent.com/37082801/76181350-24963080-6197-11ea-9456-63bc6bd489d2.gif)

Reply:
@elaihau I've resolved the issue, there needs to be a certain mapping between vscode preference names and their xterm counterparts. Since we previously didn't handle these mappings during the `onPreferenceChanged` event the terminals were not successfully updated.